optimum/exporters/neuron/config.py

```diff
@@ -53,3 +56,39 @@ class TextAndVisionNeuronConfig(NeuronConfig):
     """
 
     DUMMY_INPUT_GENERATOR_CLASSES = (DummyTextInputGenerator, DummyVisionInputGenerator, DummyBboxInputGenerator)
+
+
+class DecoderNeuronConfig(ExportConfig):
+    """
+    Handles decoder architectures.
+    """
+
+    NEURONX_ARGS = {"batch_size": 1, "n_positions": 128, "tp_degree": 2, "amp": "f32"}
+    NEURONX_MODULE = None
+    NEURONX_CLASS = None
+
+    def __init__(self, task, **kwargs):
+        if not is_transformers_neuronx_available():
+            raise ModuleNotFoundError("The transformers-neuronx package is required.")
+        module = importlib.import_module(self.NEURONX_MODULE)
+        self._neuronx_class = getattr(module, self.NEURONX_CLASS, None)
+        if self._neuronx_class is None:
+            raise ImportError(f"{self.NEURONX_CLASS} not found in {self.NEURONX_MODULE}. Please check versions.")
+
+    def split_kwargs(self, **kwargs):
+        """Split between kwargs that need to be passed when loading the transformers model
+        and those that need to be passed to the neuron optimizer.
+        """
+        model_kwargs = kwargs
+        neuron_kwargs = {}
+        for arg, default in self.NEURONX_ARGS.items():
+            if arg in model_kwargs:
+                neuron_kwargs[arg] = model_kwargs[arg]
+                model_kwargs.pop(arg)
+            else:
+                neuron_kwargs[arg] = default
```

Comment:
Actually, they will never be any unused `kwargs`: the remaining `model_kwargs` will all be passed to `get_model_from_task`, and it will raise an error if one of them is not supported.
I think that the `modeling_base.py` export code ignores the model `kwargs` when calling `get_model_from_task`, but I cannot do that because some of them are really important like `low_cpu_memory`.